app/src/main/res/values/strings.xml

```diff
@@ -20,7 +20,7 @@
     <!-- This is used as the label on a button on a first-run welcome screen to sign in and start using the application -->
     <string name="welcome_start_btn">Get Started</string>
     <!-- This provides context near a button on the first-run welcome screen to describe application requirements. Firefox Account is to be translated consistently to match use in other Firefox products. %1$s will be replaced with application name. -->
-    <string name="welcome_instructions">To use %1$s, you’ll need a\nFirefox Account with saved logins.</string>
+    <string name="welcome_instructions">To use %1$s, you’ll need a Firefox\nAccount with saved logins.</string>
```

Comment:
This indicates that we want to have on-device testing for this string. Automated screenshot testing goes a long way on problems like this. CC @Delphine to point at other engineers that have done so successfully.

Reply:
@npark-mozilla has been helping us for a long time with screenshots on Android. And I don't know anyone on Lockbox side that has been involved with this, but maybe No-Jun can share some knowledge?

Reply:
@Delphine so in other projects, we have been using fastlane tools and espresso UI test script that goes through the screens and generate l10n screenshots.  It has it's quirks, but it does scale well if the number of screenshots increase (also easy to switch locales).  Only caveat is that it does not work with kotlin, so the espresso test must be written in Java.

Reply:
Good call out here, thanks.

After discussing with @sashei and @eliserichards we will not use a newline and instead explore  some layout constraints to get closer to the intended look (and line wrapping).